src/utils/io/spi.cpp

```diff
@@ -28,35 +28,35 @@
 #include <unistd.h>
 #include <sys/mman.h>
 
-#ifndef WIN
-#include <linux/spi/spidev.h>
+#if LINUX
+    #include <linux/spi/spidev.h>
```

Comment:
Preprocessor directives should not create indentation levels, as per our [styleguide](https://hyp-ed.github.io/styleguide/#Preprocessor_Directives).

Reply:
Thanks for pointing that out, will fix, didn't even know that was in our style guide.

Reply:
Actually can we just remove this from our style guide? The [reasons](https://stackoverflow.com/a/789156/7162381) for not indenting preprocessor directives seem to be quite outdated and don't even apply to us (unless there's something I'm overlooking). Imo it's more readable when indented.

Reply:
Please, raise an issue on the styleguide repo and we will vote :)

Reply:
Haha fair, I'll put up an issue after this pr.